packages/ui/lib/components/VisualizerOverlay/index.tsx

```diff
@@ -21,32 +23,65 @@ const VisualizerOverlay: React.FC<VisualizerOverlayProps> = ({
   exitFullscreenLabel,
   isFullscreen = false
 }) => {
-  const presetOptions = presets.map(preset => ({
+  // State for managing overlay opacity
+  const [isHovered, setIsHovered] = useState(false);
+
+  // Establish timer for overlay opacity
+  useEffect(() => {
+    let timeoutId: number | undefined;
+    if (isHovered) {
+      // Set the timeout to remove the hover effect after 3 seconds
+      timeoutId = window.setTimeout(() => {
+        setIsHovered(false);
+      }, 3000);
+    }
+    return () => {
+      if (timeoutId !== undefined) {
+        clearTimeout(timeoutId);
+      }
+    };
+  }, [isHovered]);
+
+  // Handler definitions for relevant mouse events
+  const handleMouseEnter = () => setIsHovered(true);
+  const handleMouseMove = () => setIsHovered(true);
+  const handleMouseLeave = () => setIsHovered(false);
+
+  const presetOptions = presets.map((preset) => ({
     text: preset,
     value: preset
   }));
 
-  return <div className={styles.visualizer_overlay}>
-    <Dropdown
-      search
-      selection
-      options={presetOptions}
-      defaultValue={selectedPreset}
-      onChange={onPresetChange}
-    />
-    {
-      isFullscreen
-        ? <p>{exitFullscreenLabel}</p>
-        : (
-          <Button
-            basic
-            icon='expand'
-            onClick={onEnterFullscreen} 
-          />
-        )
-    }
-  </div>;
+  const stylesMod = classNames.bind(styles);
+
+  const ovClasses = stylesMod({
+    visualizer_overlay: true,
+    hover: isHovered
+  });
+
+  return (
+    <div
+      className={ovClasses}
```

Comment:
```suggestion
      className={classnames(styles.visualizer_overlay, {
          [styles.hover]: isHovered
      })}
```

And then you can remove lines 55-60.

I also recommend importing `classnames` as `cx` to make it shorter.